src/libmain/shared.hh

```diff
@@ -66,6 +66,30 @@ template<class N> N getIntArg(const string & opt,
     return n * multiplier;
 }
 
+template<class N> N getFloatArg(const string & opt,
```

Comment:
I think it isn't used (yet). I was not sure whether any of the code that accepts arguments on the command line that are passed into the language would use this or the regular parser.

I added it for good measure and already expected this to get questioned and if you think itis superfluous I'm more than happy to take it out. :)